etl/steps/data/garden/wb/2024-09-09/food_prices_for_nutrition.py

```diff
@@ -14,17 +14,42 @@
 # Classification to adopt (by default, the latest one).
 CLASSIFICATION = EXPECTED_CLASSIFICATIONS[-1]
 
+# All costs are given in constant 2021 PPP$ per person per day, except for the cost of a healthy diet,
+# which is reported in current PPP$ per person per day for each year.
+# To express the healthy diet costs in constant 2021 PPP$, we need to adjust for inflation.
+# This is done by multiplying the cost in a given YEAR by the ratio CPI(BASE_YEAR) / CPI(YEAR).
+# Base year for CPI corrections.
+CPI_BASE_YEAR = 2021
```

Comment:
The method looks good, but I am still not sure if it's 2021 or 2017 here. If the PPP rates used were in 2021 prices, this is fine. If they are in 2017 prices it should be 2017.